ide/extbrowser/src/org/netbeans/modules/extbrowser/NbDdeBrowserImpl.java

```diff
@@ -212,7 +271,15 @@ private String realDDEServer () {
         }
         
         try {
-            String cmd = getDefaultOpenCommand ();
+            String cmd = getDefaultOpenCommand();
+            
+            /** if not found with getDefaultWindowsOpenCommand function
+             *  fallback to previous method
+             */
+            if (cmd.isEmpty()) {
+                    cmd = getDefaultOpenCommand();
+                }
+            
```

Comment:
the same method is called twice, something is wrong here.

this is probably supposed to be:
```java
            String cmd = getDefaultWindowsOpenCommand();
            if (cmd == null || cmd.trim().isEmpty()) {
                cmd = getDefaultOpenCommand();
            }
```
if the fallback is still required needs to be discussed. My feeling is that it does the same thing just via native code, if true, it should be removed in my opinion since it would be just a redundant call giving the same results.

Reply:
I think I was in to much of a rush to resubmit the pull request that I managed to missed this

The previous method tries to work out the default browser by looking in HKEY_CLASSES_ROOT at .html, the new method finds it by looking in HKEY_CURRENT_USER at the userChoice for https.